### PR TITLE
salinity Issue 345

### PR DESCRIPTION
**Pull request by kaiiam**
Addressing https://github.com/pato-ontology/pato/issues/345, add an ID range for myself and add salinity as discussed.

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -21836,6 +21836,14 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070001
+name: salinity
+def: "An osmolarity quality inhering in a bearer by virtue of the total amount of ionic salts dissolved in the bearer." [http://db0nus869y26v.cloudfront.net/en/Salinity, https://en.wikipedia.org/wiki/Salinity, https://rwu.pressbooks.pub/webboceanography/chapter/5-3-salinity-patterns/, https://www.sciencedirect.com/referencework/9780080983004/treatise-on-geochemistry, https://www.sciencedirect.com/science/article/pii/B9780857096807500012]
```
Comment (nicolevasilevsky) [inline src/ontology/pato-edit.obo:21842]: If you are referring a book, I think the preferred format is to xref the ISBN in the format ISBN:XXXXXXXXXXX

Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo:21842]: otherwise this looks good to me

Comment (sbello) [reply src/ontology/pato-edit.obo:21842]: Agree with Nicole. Switch the one book to ISBN and then this is good to go.

**Review by sbello — CHANGES_REQUESTED** [review]

**Comment by kaiiam** [comment]
Thanks @nicolevasilevsky and @sbello for the quick review! The reference to which I believe you are referring `Introduction to Oceanography` https://rwu.pressbooks.pub/webboceanography/ is an interesting publication format, which I haven't seen before, it appears to be a completely online book through [Roger Williams University Open Publishing](https://rwu.pressbooks.pub/) as such it does not have an ISBN number. 

image

Let me know if this is OK or if you want me to find a different reference or another action. Thanks!

**Review by sbello — APPROVED** [review]

**Comment by kaiiam** [comment]
Great thanks @sbello, @nicolevasilevsky and or @dosumis let me know if you get a chance to review this.

**Comment by kaiiam** [comment]
Looks like the commit https://github.com/pato-ontology/pato/commit/c3cb5977036df737b8d1d9200f61130aa1d26930 added id-ranges 15-17, which is causing a merge conflict with this request, as I had added range myself in id range 15. I'll update this PR to add myself in id range 18 and update the ID for the newly requested salinity term.

**Comment by kaiiam** [comment]
@dosumis I though I had figured it regarding the merge conflict out, the code looked right but it's still registering as a merge conflict even though the PR now just adds my ID range. Clicking on the `resolve conflict` button I see the conflict as:

image
which is just the new changes. 

@dosumis Please let me know if messed something up here or what the next steps are. I though running `git checkout upstream/master pato-idranges.owl` would bring it up to date.

**Comment by dosumis** [comment]
Looks like git is just being liberal in flagging clashes.  Should be sufficient to manually remove diff tags.

**Comment by kaiiam** [comment]
@dosumis since I don't have write permission I don't believe I am able to do that would it be possible for you to?

**Comment by matentzn** [comment]
@kaiiam I made you an editor with write access. :) I trust you wont delete PATO with your newfound powers.

**Comment by kaiiam** [comment]
Thanks @matentzn I think I've fixed the issue at hand. 

Thanks for making me an editor, I promise not to do anything wonky, just regular PRs via branches.

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by kaiiam** [comment]
@sbello @dosumis @matentzn please let me know if any of you could take a moment to review this. I believe it was previously approved by @sbello prior to the merge conflict caused by https://github.com/pato-ontology/pato/commit/c3cb5977036df737b8d1d9200f61130aa1d26930. 

Much obliged!

**Review by matentzn — APPROVED** [review]

**Pull request merged**